js/scenes/Compass.js

```diff
@@ -56,17 +80,27 @@ export default class Debug extends Component {
             toLat={51.504263}
             toLon={-0.088266}/>
         </View>
+        {this.renderLocation(this.props.geolocation)}
       </View>
```

Comment:
I think it's easier to read if you have the main `render()` function fist and 'sub-render' function (`renderLocation()`) after. So you get the high level render content first and scroll down to dig into detail. Maybe just me though, but this is a pattern I've been workign to.